### PR TITLE
the revenge of the captain's beret

### PR DESCRIPTION
**Pull request by Addust**
the html test is something i got bored and wanted to do anyways

## About The Pull Request

Captains now have a beret inside their garment bag.

## Why It's Good For The Game

it has been lying untouched in the code for too long. i hereby restore captain beret priveleges

## Changelog
:cl:
fix: CentCom remembered where they kept all the Captain berets, and have actually started stocking them in garment bags.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/garment.dm [hunk 1]
```diff
@@ -61,6 +61,7 @@
 	new /obj/item/clothing/gloves/captain(src)
 	new /obj/item/clothing/head/costume/crown/fancy(src)
 	new /obj/item/clothing/head/hats/caphat(src)
+	new /obj/item/clothing/head/caphat/beret(src)
```
Comment (distributivgesetz) [inline code/game/objects/items/storage/garment.dm:64]: the beret needs to be repathed to `/obj/item/clothing/head/hats/caphat/beret` else it won't inherit the armor from `/obj/item/clothing/head/hats/caphat`

**Review by strandsofivy — CHANGES_REQUESTED** [review]

File: code/game/objects/items/storage/garment.dm (hunk 1 above)
Comment (Addust) [reply code/game/objects/items/storage/garment.dm:64]: i used that path but it caused test failures

Comment (distributivgesetz) [reply code/game/objects/items/storage/garment.dm:64]: you need to update the item definition too

Comment (Addust) [reply code/game/objects/items/storage/garment.dm]: done

**Comment by LemonInTheDark** [comment]
Beret creep. why can we not just fucking add unique gear instead of giving the same "neat" hat to every single job on the station.
Justify this

**Comment by Thunder12345** [comment]
Stop giving everyone berets it's getting so old Indiana Jones tried to put it in a museum

**Comment by Addust** [comment]
> Beret creep. why can we not just fucking add unique gear instead of giving the same "neat" hat to every single job on the station. Justify this
> 
> (I realize this is likely just "I saw unused hat" but this really bothers me and I need something more then guil leaving it in to be implemented in future)

it is "i saw unused hat" and i'd rather not see it go to waste, if a captain wants the extra option they can have the extra option. some people like berets, some like caps, some like whatever funky scifi helmet design their job gets issued by default. its a matter of personal taste, and removing choice is not what should be done

**Comment by Addust** [comment]
> Stop giving everyone berets it's getting so old Indiana Jones tried to put it in a museum

and yet i continue to space explore

**Comment by Serxule** [comment]
why do so many people cry about berets when they're used by multiple people every round

**Comment by Addust** [comment]
> why do so many people cry about berets when they're used by multiple people every round

THIS

this is the last voice of reason in /tg/station

**Comment by BlueMemesauce** [comment]
The captain should absolutely have a beret, he's the leader of the station and berets are often associated with commanders. Unlike giving berets to atmos techs, it's not beret creep at all to give the captain a beret.

**Comment by DaydreamIQ** [comment]
If people are gonna complain about beret creep then what about the fact that every department gets a bandanna and winter coat? Just let us have our customization options christ.

**Comment by GuillaumePrata** [comment]
> (I realize this is likely just "I saw unused hat" but this really bothers me and I need something more then guil leaving it in to be implemented in future)

Wow, I don't mind you mixing me with Golden, but on the Beretcreep issues???
Gonna make a PR removing all the beret armor values out of vengeance now!

**Comment by GuillaumePrata** [comment]
Memery aside and repeating what I already said on Discord.

The captain already has one, if not the most iconic hats in the game already.

Giving them another boring beret is just a massive downgrade.
Berets should have stayed as Security/Mime headwear so they stay as an iconic hat and work as identity for these jobs.

**Comment by LemonInTheDark** [comment]
> > (I realize this is likely just "I saw unused hat" but this really bothers me and I need something more then guil leaving it in to be implemented in future)
> 
> Wow, I don't mind you mixing me with Golden, but on the Beretcreep issues??? Gonna make a PR removing all the beret armor values out of vengeance now!

NOT AGAIN

**Comment by CRITAWAKETS** [comment]
i'll be honest the captain's beret just fit the most situations when I used it. Like you'd expect a beret to not fit with anything, but in this case it actually worked a lot better than the other hats.

**Comment by Addust** [comment]
> i'll be honest the captain's beret just fit the most situations when I used it. Like you'd expect a beret to not fit with anything, but in this case it actually worked a lot better than the other hats.

i myself am a captain beret fan and have used it well on yogstation and i can attest as to the correctness of this statement

**Comment by moocowswag** [comment]
If the pr isnt merged just find another beret and try to spraypaint it blue.

**Comment by ninjanomnom** [comment]
Go beat up a mime to steal their beret, clothing exclusivity is a good thing.

**Comment by DrDiasyl** [comment]
> Go beat up a mime to steal their beret, clothing exclusivity is a good thing.

_Or you can buy beret at really any clothing vendor on the station._
Doesn't sound really exclusive.

**Comment by BlueMemesauce** [comment]
> Go beat up a mime to steal their beret, clothing exclusivity is a good thing.

Mime berets and military berets, although technically the same type of clothing, are worn differently and should have different sprites. Maybe add a common military beret for security/command if you're concerned about too many berets? Or maybe make the way you wear it adjustable?

**Pull request closed without merging**